bridge/client/app/app.routing.ts

```diff
@@ -94,6 +94,8 @@ const routes: Routes = [
               { path: '', pathMatch: 'full', redirectTo: 'project' },
             ],
           },
+          { path: 'environment', component: KtbEnvironmentViewComponent },
```

Comment:
I would also add a redirect here. `project/${projectName}` should be redirected to `project/${projectName}/environment`. @ermin-muratovic, @renepanzar  what's your opinion on that?

Reply:
@Kirdock , @ermin-muratovic - you're more experienced with the project, what are your thoughts?

Reply:
Maybe it's a bit unclear.
- At the moment the environment screen is `project/${projectName}`.
- With the additional params we introduced `project/${projectName}/environment/...` in order to not interfere with other routes. `project/${projectName}/environment/` did not exist before, so there aren't any links to it.
- It is a weird behavior if you go to the environment screen (`project/${projectName}`), select a stage, and then `/environment/...` is added to the path

Reply:
To be honest, I'm not sure if anyone is paying attention to the URL when using the app. With this in mind I wouldn't spend too much time thinking about this "weird behavior". If it works, it works :)